homeassistant/components/brother/sensor.py

```diff
@@ -66,10 +81,30 @@ def state(self):
     def device_state_attributes(self):
         """Return the state attributes."""
```

Comment:
What do you think of introducing two local variables `remaining_pages` and `drum_counter` to save the relevant const?

```suggestion
        """Return the state attributes."""
        remaining_pages  = None
        drum_counter = None
```